tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -42,7 +57,7 @@ CREATE TABLE IF NOT EXISTS genesis_assets (
 
     asset_tag TEXT UNIQUE NOT NULL,
 
-    meta_data BLOB,
+    meta_data_id INTEGER REFERENCES assets_meta(meta_id),
```

Comment:
Is the meta field optional? I think it's not on the Golang side. So the question would be, would we actually insert `NULL` here if the `MetaHash` array is a zero array? I'd say we probably should just insert the zero-hash as a meta record and make this field `NOT NULL`.

Reply:
I think to match the previous behavior, either this field can be NULL or the `meta_data_hash` can be NULL.

Reply:
Yes, it's optional so a NULL can be inserted here. This would mean the asset has no meta data associated with it, and the `metaHash` is all zeroes. 

We wouldn't explicitly insert anything, since this uses the `NullInt` type on the Go side. The all zeroes meta has no meaning, so I don't think we need to insert it. Queries that determine if an asset has a meta or not, would check for NULL there, or do a join, which would return empty rows.

Reply:
Ok so with the way things are rn: we always insert the "all zero" meta on disk. As a result, anything that doesn't have a meta, will end up pointing to that (dedup).

Reply:
In that case can't this be `NOT NULL` now?